src/lib/components/Grid.svelte

```diff
@@ -47,7 +47,7 @@
 	$: visibleItems = items.slice(0, visibleNumber);
 </script>
 
-<div class="grid grid-cols-[repeat(auto-fit,minmax(200px,1fr))] gap-4 pt-8 {className}">
+<div class="grid grid-cols-[repeat(auto-fit,minmax(300px,1fr))] gap-4 pt-8 {className}">
```

Comment:
this wasn't an error? I wanted to make the grid a bit thinner. You find it better when it's 300px wide?